src/components/StyleGuide/DesignTokens.stories.mdx

```diff
@@ -0,0 +1,170 @@
+| 5. [Using NYPL Design Tokens in Javascript](#using-nypl-design-tokens-in-javascript) |
+| 6. [NYPL Design Token Categories](#nypl-design-token-categories)                     |
+
+## What are Design Tokens?
+
+Design tokens are the style values that define a design system. These values can include, but may not be limited to, spacing, color, typography, breakpoints and object styles. Ultimately, these values are used to construct and maintain the components that make up our system.
+
+## Why Use Design Tokens?
+
+_"With design tokens, you can capture low-level values and then use them to create the styles for your product or app. You can maintain a scalable and consistent visual system for UI development."_ – Jina Bolton, Lead Designer for the Salesforce Design System
+
+Design tokens are a single source of truth when it comes to styles. They allow an organization to more easily build multiple applications – web and/or native – that feel and look the same without slowing down the development team.
+
+## How to Use the NYPL Design Tokens
+
+The NYPL design tokens are built into the NYPL Design System (the DS) and are offered in two formats for consumption: CSS variables and javascript data objects. To access either of these formats, the DS v0.25.2 or higher needs to be properly installed in a consuming React application.
+
+## Using NYPL Design Tokens in CSS
+
+If your app uses CSS or SCSS to handle styles, the DS creates standard CSS custom properties (e.g. variables) for consumption in your stylesheets. If the DS is properly installed, the CSS custom properties will be available for you to use. Nothing more needs to be done to your app.
+
+To assist with CSS scope isolation, all of the CSS custom properties created by the DS are prefixed with "nypl." Please see the examples below.
+
+```css
+--nypl-colors-brand-primary
+--nypl-colors-brand-secondary
+--nypl-fontSizes-sm
+--nypl-fontSizes-md
+--nypl-fontSizes-lg
+--nypl-fontSizes-xl
+--nypl-space-xxs
+--nypl-space-xs
+--nypl-space-s
+--nypl-space-m
+--nypl-space-l
+```
+
+The CSS custom properties are defined on the :root pseudo-class and should be implemented using standard CSS custom properties conventions.
+
+```css
+/* property declaration */
+:root {
+  --nypl-colors-brand-primary: #c60917;
+}
+
+/* property usage */
+.custom-button {
+  background: var(--nypl-colors-brand-primary);
+}
+```
+
+## Using NYPL Design Tokens in Javascript
+
+If you would like to use CSS-in-JS styles in your React components, the `useNYPLTheme` hook will provide you with NYPL design token values. The hook depends on the `DSProvider` component and will not work properly outside of the `DSProvider` wrapper component.
+
+After importing and implementing the `DSProvider` wrapper component, as explained in the `DSProvider` section of the [DS Chakra UI](?path=/story/chakra-ui--page#dsprovider) document, your children components can use this hook function.
```

Comment:
Will this URL work in production? It works in the Tugboat dev URL but it may not in production. If it does, great. That'll help update other URLs we have in our Storybook instances.

Reply:
I don't know if this will work on the prod site.  Truthfully, at this point, I am unsure that the best method is for linking from one page to another within Storybook.  If you don't have another recommendation, let's go with this.  If it ends up not working on prod, we can fix it up in the next release.